packages/babel-generator/test/fixtures/types/ImportAssertion/options.json

```diff
@@ -1,9 +1,9 @@
 {
   "plugins": [
     [
-      "moduleAttributes",
+      "importAssertions",
       {
-        "version": "may-2020"
+        "version": "september-2020"
```

Comment:
The `import-assertions` is stage 3 now. I expect the syntax is now stable for implementation thus a `"version"` key seems a bit verbose to me.